Move execution-environments/repositories endpoints to v3

jerabekjiri opened the pull request:
#### What is this PR doing:
<!-- Describe your changes giving context and all the needed details. -->

<!-- Add Jira issue link -->
Issue: [AAH-766](https://issues.redhat.com/browse/AAH-766)

This PR changes only `execution-environments/repositories` routes and its sub-routes. 
Changed URLs (moved from `_ui/v1/execution-environments/` to `v3/plugin/execution-environments`: 
- `/api/automation-hub/v3/plugin/execution-environments/repositories/`
- `/api/automation-hub/v3/plugin/execution-environments/repositories/testcontainer/_content/images/`
- `/api/automation-hub/v3/plugin/execution-environments/repositories/testcontainer/_content/readme/`
- `/api/automation-hub/v3/plugin/execution-environments/repositories/testcontainer/_content/history/`
- `/api/automation-hub/v3/plugin/execution-environments/repositories/testcontainer/_content/tags/`
- `/api/automation-hub/v3/plugin/execution-environments/repositories/testcontainer/_content/sync/`

- added/changed attributes to `id, pulp_href, created_at and updated_at` according to issue description
- added rbac tests:
    - `ee_namespace_list_roles`
    - `ee_namespace_my_permissions`
    - `ee_namespace_add_role`
    - `ee_namespace_remove_role`

Deleted URLs: 
-`/api/automation-hub/_ui/v1/execution-environments/namespaces`


#### Reviewers must know:
<!-- e.g: Testing steps, dependencies, needed branches etc. -->

#### Notes: 

**PR Author**: Add a QE reviewer ([exceptions](https://docs.engineering.redhat.com/display/AUTOHUB/Other+Team+Processes#OtherTeamProcesses-galaxy_ngrepo)); 
**Reviewers**: look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

newswangerd reviewed (commented):
This looks very good! I haven't seen any major changes that need to be made so far. I want to test this with the UI, and then I'll sign off on it.

jerabekjiri commented on galaxy_ng/app/viewsets.py:
```diff
@@ -57,11 +60,6 @@ class NamespaceViewSet(
     endpoint_name = "pulp_ansible/namespaces"
 
 
-# This is here because when new objects are created, pulp tries to look up the viewset for
```
oops :upside_down_face:, reverted

jerabekjiri commented:
@newswangerd I had to do a little refactoring in `test_ui_paths` and moved new `repositories` tests to the new `test_v3_plugin_paths` file. Could you please double-check that I refactored it correctly [974efda](https://github.com/ansible/galaxy_ng/pull/1397/commits/974efda92e5c0f2c38c55939a7d62a54f42bd8db). I switched to `api_client` and it acts little differently.
